.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea/
```

Comment:
Not sure adding this is a good idea -- I'd use my global .gitignore for this.

Reply:
Yes, if nothing else this change doesn't belong in this PR.